### ISSUE TITLE
Feature Request: Render Matrix support

### ISSUE THREAD

**Issue by nikolalkc** [karltechno/Reaper-Waapi-Transfer#17]
It would be awesome if waapi transfer could detect selected regions in region matrix without clicking Add to render queue. Render time would be faster because no temp project would have to be created and using this [JerContact_Open project based on file in clipboard.lua](https://forum.cockos.com/showthread.php?t=207574) would also be possible. I don't know how hard implementing this feature would be, but this is definetely a game changer.

**Comment by karltechno** [comment]
Yeah I had planned to do something similar when I had more time to work on the extension - I can appreciate how useful it would be. I will try and take another look when I have some more time and see what I can do.

**Comment by nikolalkc** [comment]
That's great news. I can't wait for it to happen :) Good luck.
